external-crates/move/crates/move-package/tests/test_sources/diamond_problem_dep_incorrect_dev_override/Move@resolved.snap

```diff
@@ -2,7 +2,7 @@
 source: crates/move-package/tests/test_runner.rs
 ---
 When resolving dependencies for package Root, conflicting versions of package C found:
+At C
```

Comment:
Hmm... not sure about this change.

Reply:
It is a result of changing the order of `existing_pkg_info`